functional_tests/mp/01-2-simple-no-finish.yea

```diff
@@ -0,0 +1,35 @@
+id: 0.mp.01-2-simple-no-finish
+plugin:
+  - wandb
+tag:
+  shard: service
+  skip: True
```

Comment:
Please add a "TODO: rm me" comment to unskip this eventually.

Reply:
it is mentioned in this ticket: https://wandb.atlassian.net/browse/WB-9158